### ISSUE TITLE
Fetching feature should fetch images from the camera

### ISSUE THREAD

**Issue by soravux** [lvsn/CameraNetwork#9]
Previously, all images end up in the raspberry Pi and the fetching feature loads them from there. Now, the images are all on the Camera's SD card. The fetching from there must be done before the fetching to the server.

**Comment by soravux** [comment]
Commit d106a39b8e567e1007e1228f73d86ef2b93f70cb added scripts that renames and organizes the files in a folder hierarchy based on EXIF data (exiftool and Python needed).

**Comment by MathGaron** [comment]
as a first step toward a proper download, I added the possibility to get camera's memory information (wich may be usefull for the user before downloading or for the system to know when to download maybe). I can work on this issue if its ok with you?

for the script, is it launched from the app or we have to launch it by ourself?
- if you have suggestion or idea dont hesitate!

**Comment by soravux** [comment]
@MathGaron : Go for it! :)
The only thing I though was about making small batches of downloads to the pi (~25-50 images?) and download them from victoria using rsync.

For now, the script have to be launched manually. If CameraNetwork could execute if after a successful fetch, that woul be nice!

**Comment by MathGaron** [comment]
Ok I will look for rsync! Would it be a good feature to show on the gui how many images are in the camera's memory?

**Comment by jflalonde** [comment]
Duplicate of #16
